package-lock.json

```diff
@@ -120,9 +120,6 @@
     },
     "@existentialism/package-b": {
       "version": "file:packages/package-b",
-      "requires": {
-        "@existentialism/package-a": "file:packages/package-a"
```

Comment:
☝️ should be kept

Reply:
Thanks for putting this together! 😍 will take a look!